examples/blowout_wake/inputs_SI

```diff
@@ -4,6 +4,11 @@ hipace.predcorr_max_iterations = 1
 hipace.predcorr_B_mixing_factor = 0.12
 hipace.predcorr_B_error_tolerance = -1
 
+my_constants.kp_inv = 10e-6
+my_constants.kp = 1 / kp_inv
+my_constants.wp = clight * kp
+my_constants.ne = wp^2 * m_e * epsilon0 / q_e^2
```

Comment:
True, but no dot also works because the parser always uses double internally.

` blowout_wake/inputs_ionization_SI` calculates `kp_inv` from `ne`. Should another file do this too?

Reply:
Let us keep this discussion for later. I think that `examples/blowout_wake/inputs_SI` is the first example people would look at, because it is the most usual use case, so we should make it meaningful. But let's keep it as is for now.